cli/command/container/cp.go

```diff
@@ -68,9 +102,12 @@ func NewCopyCommand(dockerCli command.Cli) *cobra.Command {
 		},
 	}
 
+	fileInfo, _ := os.Stdin.Stat()
+
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.followLink, "follow-link", "L", false, "Always follow symbol link in SRC_PATH")
 	flags.BoolVarP(&opts.copyUIDGID, "archive", "a", false, "Archive mode (copy all uid/gid information)")
+	flags.BoolVarP(&opts.quiet, "quiet", "q", (fileInfo.Mode()&os.ModeCharDevice) == 0, "Does not display copying process")
```

Comment:
I think `dockerCli` has a `IsTerminalOut` function (which is what we need to check here).

Reply:
A quick Cmd+F for `IsTerminalOut` in project results nothing 😕

Reply:
Found it: `dockerCli.Out().IsTerminal()`

Reply:
Alright!

Reply:
Now yes @cpuguy83